_posts/2023-05-25-Announcing-Data-Prepper-2.3.0.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: post
+title:  "Announcing Data Prepper 2.3.0"
+authors:
+- dlv
+- kkondaka
```

Comment:
I've been following the pattern of putting our names alphabetical ordered by last name.